packages/web/src/components/ui/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './Pagination.module.scss';
+import clsx from 'clsx';
+
+const Pagination = () => {
```

Comment:
Not sure if we gonna need the pagination component, but if we are, probably we could try to parallelize our work and ask you to implement it completely. @spalt08 what do you think?